CONTRIBUTING.md

```diff
@@ -17,14 +17,18 @@ To test with a real GitHub repository, you'll need to create a test repository a
   - **Secret:** `development`
   - **Which events would you like to trigger this webhook?:** Choose **Send me everything**.
 0. Create a `.probot.yml` in your repo with:
+        ```
```

Comment:
I think this is an older version. I fixed it in https://github.com/bkeepers/PRobot/pull/38/commits/bbd3f7cf1326ba4befbf7a44a8c4829a66b30d34